ClinGen/clincoded
# Connects to #941. VCI Basic Info tab's SCV table.

**Steps to test:**
1. Go to VCI and select **17000** as the ClinVar ID for testing.
2. Upon proceeding to the **Basic Info** tab, expect to see the SCV table with a title **ClinVar Assertions** and a external linkout to **ClinVar** in the table header.
3. Click on any of the external linkouts in the SCV table and expect to see the target destination to be opened in a new tab/window or the target PDF to be opened (or downloaded).
4. In the **Submitter - Study name** column of the SCV table, click on the **Study description** links and expect to be able to toggle the visibility of the PopOver UI displaying the descriptive text.
